ledger/src/blocktree_db.rs

```diff
@@ -35,6 +35,8 @@ const INDEX_CF: &str = "index";
 const DATA_SHRED_CF: &str = "data_shred";
 /// Column family for Code Shreds
 const CODE_SHRED_CF: &str = "code_shred";
+/// Column family for Code Shreds
```

Comment:
nit: copy-pasta

Reply:
how did that possibly get there? I blame the 🐈